test/EasyRdf/GraphTest.php

```diff
@@ -167,7 +167,7 @@ public function testParseFileRelativeUri()
 
         $doc = $graph->get('foaf:PersonalProfileDocument', '^rdf:type');
         $this->assertClass('EasyRdf\Resource', $doc);
-        $this->assertRegExp('|^file://.+/fixtures/foaf\.rdf$|', $doc->getUri());
+        $this->assertMatchesRegularExpression('|^file://.+/fixtures/foaf\.rdf$|', $doc->getUri());
```

Comment:
The assertRegExp is deprecated in PHPUnit - the polyfill allows us to use the current method when running tests on the older versions of PHPUnit.